app/controllers/notifications_controller.rb

```diff
@@ -98,16 +98,17 @@ def show
 
   def expand_comments
 
-    scope = notifications_for_presentation.newest
+    scope = original_scope = notifications_for_presentation.newest
     scope = load_and_count_notifications(scope) unless request.xhr?
 
     ids = scope.pluck(:id)
     position = ids.index(params[:id].to_i)
-    @notification = current_user.notifications.find(params[:id])
+    @notification = original_scope.find(params[:id])
```

Comment:
Looks like this change is an accidental revert from a merge?

Reply:
possibly yeah, I had a bit of a sketchy merge couple fo days back.